tablecloth/decoration.cpp

```diff
@@ -237,7 +237,7 @@ void main()
 
     auto Context::render_decorations(View& view, RenderData& data) -> void
     {
-      if (view.maximized || view.wlr_surface == nullptr) {
+      if (view.maximized || view.wlr_surface == nullptr || !view.deco.is_visible()) {
```

Comment:
I think in most cases shadows should be drawn even when no decorations are used

Reply:
GTK draws its own shadows! This results in very ugly double shadows!